test/metric_learn_test.py

```diff
@@ -120,6 +120,15 @@ def test_iris(self):
     csep = class_separation(rca.transform(), self.iris_labels)
     self.assertLess(csep, 0.25)
 
+  def test_feature_null_variance(self):
+    rca = RCA_Supervised(dim=2, pca_comps=3, num_chunks=30, chunk_size=2)
+    X = np.hstack((self.iris_points,
+        np.eye(self.iris_points.shape[0], M = 1)))
+    rca.fit(X, self.iris_labels)
+    csep = class_separation(rca.transform(), self.iris_labels)
+    self.assertLess(csep, 0.30)
+
+
```

Comment:
We should also have a test using `pca_comps=0.95`. It can use the same `X` matrix and check the same result.